pkgs/development/python-modules/pyvows/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, gevent
+, preggy
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "pyvows";
+  version = "3.0.0";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    pname = "pyVows";
+    inherit version;
+    sha256 = "sha256-2+4umWLNkbFlCpfFwX0FA2N0zOZhst/YM4ozBfXoaMI=";
+  };
+
+  propagatedBuildInputs = [
+    gevent
+    preggy
+  ];
+
```

Comment:
Could you please add `checkInputs = [ pytestCheckHook ];` to enforce running tests

Reply:
When adding that line, the build fails:

```
error: error: builder for '/nix/store/wnp51qxjdghr4hi1xasaka0bijki31gm-python3.10-pyvows-3.0.0.drv' failed with exit code 5;
              last 10 log lines:
              > Executing pythonImportsCheckPhase
              > Check whether the following modules can be imported: pyvows
              > pytestCheckPhase
              > Executing pytestCheckPhase
              > ============================= test session starts ==============================
              > platform darwin -- Python 3.10.6, pytest-7.1.2, pluggy-1.0.0
              > rootdir: /private/tmp/nix-build-python3.10-pyvows-3.0.0.drv-0/pyVows-3.0.0
              > collected 0 items                                                              
              >
              > ============================ no tests ran in 0.01s =============================
              For full logs, run 'nix log /nix/store/wnp51qxjdghr4hi1xasaka0bijki31gm-python3.10-pyvows-3.0.0.drv'.
```

This project does not use pytest for testing, instead, it uses itself for testing: 

https://github.com/heynemann/pyvows/blob/5b0e2a202603c1fc00d1fa0c6134c92c15b7e2b7/Makefile#L1

However, this Makefile is not included in the pypi package. Should I use `fetchFromGitHub` instead or should I replicate the command performed in the Makefile?